worker/contributors.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Instructions: Once you make a contribution to the repo, add yourself to this map,
+ * where the key is your user id in stacker.news prod,
+ * and the value map contains your GitHub username, and your SN nym.
+ * Right now, the GitHub usernames and SN nyms are not used, but maybe we'll use them
+ * in the future for a fancier UI decoration.
+ */
+const contributorMap = {
```

Comment:
Was thinking about pinging all persons mentioned in https://github.com/stackernews/stacker.news/graphs/contributors

but maybe it's better to just announce this on SN when it's released and then we can add them or they create a PR :thinking:

Reply:
I like the idea of them opening a PR to add themselves. A little bit of proof of work to discover their user ID, plus it makes it opt-in from the get-go